docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/04_input_controls.mdx

```diff
@@ -314,19 +314,7 @@ const selectInputUiSchema: UiSchema = {
 
 The connected multi select renderer connects to your back end in the same way as the connected select renderer. 
 
-To invoke the connected multi select renderer, in the `JSONSchema`, specify the `type` property to be `array`.
-
 ```ts
-const connectedMultiSelectJsonSchema: JSONSchema7 = {
-  type: "object",
-  properties: {
-    connectedMultiSelectInput: {
-      type: "array",
-      description: "Kotlin.String",
-    },
-  },
-};
-
```

Comment:
Why is this block being removed - do you not need the `type` property to be `"array"` anymore?